test_unstructured_api_tools/pipelines/test_convert.py

```diff
@@ -128,6 +128,14 @@ def tests_notebook_to_script(sample_notebook):
         ),
         ("def pipeline_api(bad_arg,text): pass", "First parameter must be named text"),
         ("def pipeline_api(text, m_var): pass", "Default argument for m_var must be empty list"),
+        (
+            "def pipeline_api(text, response_type, m_var=[], m_var2=[]): pass",
```

Comment:
Was there a reason you added these tests?

Reply:
i thought there should be a default value for `response_type`